query_plan.go

```diff
@@ -79,6 +82,58 @@ func (n *Node) Render() string {
 	return "\n" + tree.String()
 }
 
+type RenderedTreeWithStats struct {
+	Text         string
+	RowsTotal    string
+	Execution    string
+	LatencyTotal string
+}
+
+func (n *Node) RenderTreeWithStats() []RenderedTreeWithStats {
+	tree := treeprint.New()
+	renderTreeWithStats(tree, "", n)
+	var result []RenderedTreeWithStats
+	for _, line := range strings.Split(tree.String(), "\n") {
+		if line == "" {
+			continue
+		}
+
+		split := strings.Split(line, "\t")
+		if len(split) == 1 {
```

Comment:
Just to confirm the logic, when does `len(split) == 1` become true? Only for root node?

Reply:
Yes, it should be only for the root node of `treeprint`(not `PlanNode`)

Reply:
It may also be detected by `line == "."`.

Reply:
Thanks! Sounds good.